cluster/charts/crossplane/templates/service.yaml

```diff
@@ -21,5 +21,5 @@ spec:
   ports:
   - protocol: TCP
     port: 9443
-    targetPort: 9443
+    targetPort: {{ .Values.webhooks.port | default "9443" }}
```

Comment:
K8s spec expect this targetPort to be a `Number or name of the port to access on the pods targeted by the service. Number must be in the range 1 to 65535. Name must be an IANA_SVC_NAME. If this is a string, it will be looked up as a named port in the target Pod's container ports` : 
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#serviceport-v1-core

same as above. unquote the default..

Reply:
Good catch, see above